src/nostr-post/nostr-post.ts

```
@@ -0,0 +1,835 @@
+  const safeNoteId = sanitizeText(item.noteId);
+  html.push(`<div class="embedded-post-placeholder" data-note-id="${safeNoteId}"></div>`);
+  } else {
+  if (textBuffer) {
+    html.push(`<span class="text-content">${textBuffer.replace(/\n/g, '<br />')}</span>`);
+    textBuffer = '';
+  }
+
+  switch (item.type) {
+    case 'image':
+      // Sanitize image URL
+      const safeImgSrc = sanitizeText(item.value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)